packages/ketcher-polymer-editor-react/src/data/monomers.sdf

```diff
@@ -1288,6 +1288,64 @@ false
 [R2]H
 [R3]O
 
+$$$$
+Test-6-AP-Phosphate
+  Ketcher  9272311252D 1   1.00000     0.00000     0
+
```

Comment:
I believe we can forget to delete it before production release. We need to delete it from production bouild somehow

Reply:
Aleksandr Savelev let me leave it here